ggplot.rmd

```diff
@@ -291,15 +291,15 @@ An important argument to `geom_smooth()` is the `method`, which allows you to ch
 
 ### Boxplots and jittered points {#sub:boxplot}
 
-When a set of data includes a categorical variable and one or more continuous variables, you will probably be interested to know how the values of the continuous variables vary with the levels of the categorical variable. Say we're interested in seeing how fuel economy varies within car class. We might start with a scatterplot like this:
+When a set of data includes a categorical variable and one or more continuous variables, you will probably be interested to know how the values of the continuous variables vary with the levels of the categorical variable. Say we're interested in seeing how fuel economy varies within cars that have the same kind of drivetrain. We might start with a scatterplot like this:
 
 `r columns(1, 2/3, 0.5)`
 ```{r}
 ggplot(mpg, aes(drv, hwy)) + 
   geom_point()
 ```
 
-Because there are few unique values of both class and hwy, there is a lot of overplotting. Many points are plotted in the same location, and it's difficult to see the distribution. There are three useful techniques that help alleviate the problem:
+Because there are few unique values of both drv and hwy, there is a lot of overplotting. Many points are plotted in the same location, and it's difficult to see the distribution. There are three useful techniques that help alleviate the problem:
```

Comment:
```suggestion
Because there are few unique values of both `drv` and `hwy`, there is a lot of overplotting. Many points are plotted in the same location, and it's difficult to see the distribution. There are three useful techniques that help alleviate the problem:
```

I think I now consistently put variable names in code font